examples/bri-3/src/bri/identity/bpiAccounts/api/accounts.controller.spec.ts

```diff
@@ -38,6 +38,10 @@ describe('AccountController', () => {
   let subjectAccountStorageAgentMock: DeepMockProxy<BpiSubjectAccountStorageAgent>;
   let merkleTreeStorageAgentMock: DeepMockProxy<MerkleTreeStorageAgent>;
 
+  beforeAll(async () => {
```

Comment:
works for me as a band-aid. Testing should likely have its own config file/files that set these variables among others.  I believe this work has already begun in builders.ts and testData.helper.ts